### PR TITLE
Add html task

### PR DESCRIPTION
**Pull request by joymeetme**
html task

### CODE REVIEW

**Review by adaugochi — APPROVED** [review]

**Comment by otaruMendez** [comment]
Welldone @joymeetme, please put the html file in a HTML folder.

File: submissions/ayodeleb/HTML.TASK.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <head>
+        <title>perfectfit Login Page</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width initial-scale=1.2">
+    </head>
+    <body>
+        
+<h2>Welcome to perfectfit,this is an online shopping store for shoes, we provide comfort to  
+           your feet and make you look classy and chic.
+        </h4>
```
Comment (Gbensky) [inline submissions/ayodeleb/HTML.TASK.html:12]: Hello @joymeetme the opening tag and the closing tag should be the same <h2></h2> or <h4></h4>

File: submissions/ayodeleb/HTML.TASK.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                   <label for="password">Password</label><br>
+                   <input type="password" name="password" id="password" placeholder="******" min="6" required><br><br>
+                   <input type="checkbox" name="keepme" value="">
+                   <label for="loginmode">Remember me</label>
+                   <span><a href="#Emailcode">Forget Password?</a></span><br><br>
+                   <input type="Submit" name="login" value="Login">
+                    <a href="#signupform"><span>Sign up</span></a>
+               </fieldset>
+               </form>
+            </body>
```
Comment (Gbensky) [inline submissions/ayodeleb/HTML.TASK.html:30]: You need to properly format your tags

**Comment by joymeetme** [comment]
I  made another PR on this task and it  has been approved  so i'm closing this.

**Comment by joymeetme** [comment]
i'm  closing this PR because i created another PR on the  task and it has been approved.

**Pull request closed without merging**